server/src/interfaces/asset.interface.ts

```diff
@@ -142,64 +140,59 @@ export interface AssetUpdateDuplicateOptions {
   duplicateIds: string[];
 }
 
-export type AssetPathEntity = Pick<AssetEntity, 'id' | 'originalPath' | 'isOffline'>;
+export type AssetPathEntity = Pick<AssetEntity, 'id' | 'originalPath'>;
 
 export const IAssetRepository = 'IAssetRepository';
 
 export interface IAssetRepository {
-  getAssetsByOriginalPath(userId: string, partialPath: string): Promise<AssetEntity[]>;
```

Comment:
Did you sort these alphabetically? 😅
It's a mess in the diff lol

Reply:
Maybe I should move the sorting of repos to another PR. I can't stand them unsorted :)